PFR/ForMathlib/Entropy/Kernel/MutualInfo.lean

```diff
@@ -62,22 +62,27 @@ lemma mutualInfo_congr {κ η : Kernel T (S × U)} {μ : Measure T} (h : κ =ᵐ
     rw [snd_apply, ht, snd_apply]
   rw [entropy_congr h1, entropy_congr h2, entropy_congr h]
 
-lemma compProd_assoc (ξ : Kernel T S) [IsMarkovKernel ξ]
-    (κ : Kernel (T × S) U) [IsMarkovKernel κ] (η : Kernel (T × S × U) V) [IsMarkovKernel η] :
-    map ((ξ ⊗ₖ κ) ⊗ₖ η) MeasurableEquiv.prodAssoc MeasurableEquiv.prodAssoc.measurable
+attribute [fun_prop] MeasurableEquiv.measurable
```

Comment:
Can you please move that to a new file `PFR.Mathlib.MeasureTheory.MeasurableSpace.Embedding` so that it doesn't get lost in an otherwise unrelated file? I will soon start seriously upstreaming stuff from PFR and putting things where they belong is tremendously helpful for that. Thanks!